docker/DockerImages.md

```diff
@@ -0,0 +1,25 @@
+# Docker Images #
```

Comment:
I'd recommend making this file `README.md` to make it show up without an extra click in GitHub. (Also to have an index if we want more docs here in the future.)

Reply:
Done.